app.js

```diff
@@ -1,156 +1,182 @@
 "use strict";
 
-let userName = prompt("What is your name?");
-let points = 0;
-{
-  userName = userName.toUpperCase();
-  alert("Hello and welcome "+ userName + "!");
-  points++;
-  console.log(points);
-}
+function questions(answer) {
 
-let askAge = prompt("What is my age?");
-if (askAge == 31) {
-  alert("You guessed my age correctly!");
-  points += 2;
-  console.log(points);
-} else if (askAge > 31 && askAge <= 35) {
-  alert("I'm not that old yet, but you are close.");
-  points += 1;
-  console.log(points);
-} else if (askAge < 31 && askAge >= 25) {
-  alert("I am not that young, but you are close.");
-  points += 1;
-  console.log(points);
-} else {
-  alert("You are not even close.");
+  answer = answer.toLowerCase();
+  if(answer === "yes" || answer === "y") {
+    return true;
+  } else {
+    return false;
+  }
 }
 
-let coffee = prompt("What is my favorite coffee roast? Light, medium or dark?");
-coffee = coffee.toLowerCase();
-switch (coffee) {
-  case "light roast":
-    case "light":
-    alert("That is indeed my favorite roast");
-    points += 2;
-    console.log(points);
-    break;
-  case "medium roast":
-    case "medium":
-    alert("That is an ok coffee roast");
-    points += 1;
-    console.log(points);
-    break;
-  case "dark roast":
-    case "dark":
-    alert("Dark roast is too bitter for me.");
-    break;
-  default:
-    alert(
-      `Sorry, that is not a coffee roast. Please write dark, medium or light roast!`
-    );
-}
+let question1 = prompt("Is my age 31?");
+console.log(questions(question1));
```

Comment:
If you WANT, you can replace lines 13 & 14 with:
`console.log(questions(prompt("Is my age 31?")));` 
However, the way you wrote it is much more easily readable. Good work!